python/chronos/dev/test/run-pytests.sh

```diff
@@ -60,7 +60,7 @@ python -m pytest -v test/bigdl/chronos/autots\
                     test/bigdl/chronos/data \
                     test/bigdl/chronos/simulator \
                     test/bigdl/chronos/detector \
-       -k "not test_forecast_tcmf_distributed"
+        -k "not test_ae_fit_score_unrolled"
```

Comment:
`-k "not test_ae_fit_score_unrolled and not test_forecast_tcmf_distributed"`?

Reply:
In fact, `test_forecast_tcmf_distributed` exists only in UT part1.

Reply:
I see, thx